### PR TITLE
deps: bump celestia-node to v0.11.0-rc2

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Tested successfully on a test instance that I have running. Upgraded from v0.11.0-rc1 to v0.11.0-rc2.

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Review by tuxcanfly — APPROVED** [review]
OK.

File: ops-bedrock/docker-compose-testnet.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ services:
     container_name: celestia-light-node
     user: root
     platform: "${PLATFORM}"
-    image: "ghcr.io/celestiaorg/celestia-node:v0.11.0-rc1"
-    command: celestia light start --core.ip http://consensus-full-arabica-8.celestia-arabica.com/ --gateway --gateway.addr da --gateway.port 26659 --p2p.network arabica
+    image: "ghcr.io/celestiaorg/celestia-node:v0.11.0-rc2"
+    command: celestia light start --core.ip consensus-full-arabica-8.celestia-arabica.com/ --gateway --gateway.addr da --gateway.port 26659 --p2p.network arabica
```
Comment (tuxcanfly) [inline ops-bedrock/docker-compose-testnet.yml:19]: This still tries to resolve the DNS until I remove the trailing slash.

Comment (jcstein) [reply ops-bedrock/docker-compose-testnet.yml:19]: resolving in new PR

Comment (jcstein) [reply ops-bedrock/docker-compose-testnet.yml:19]: https://github.com/celestiaorg/optimism/pull/139

**Pull request merged**